rails/empty-arrays-in-json-request.md

```diff
@@ -0,0 +1,74 @@
+Rack::Test defaults to URL Encoding Parameters
```

Comment:
Should this be a level 1 header?

Should `Rack::Test` be wrapped in backticks?